### ISSUE TITLE
Gene annotation track appears to clear canvas incorrectly before re-rendering

### ISSUE THREAD

**Issue by ngehlenborg** [higlass/higlass#281]
_Version_:
```
SERVER_VERSION: 1.7.0
WEBSITE_VERSION: 0.6.26
LIBRARY_VERSION: 1.0.1
MULTIVEC_VERSION: 0.1.10
HGTILES_VERSION: 0.2.2
CLODIUS_VERSION: 0.9.1
TIME_INTERVAL_TRACK_VERSION: 0.1.8
```

#### Steps to reproduce
Option 1: Change colors of gene annotation track.
Option 2: Zoom out to genome wide overview, then zoom into a region and zoom out again.

#### Observed behavior
- Option 1: Original markers are overplotted (issue with transparency) and new markers appear to be shifted.

![image](https://user-images.githubusercontent.com/1216518/42421977-040ad844-82a4-11e8-82fe-382e51ecfec7.png)

- Option 2: Markers are much denser in region that was zoomed into.

![image](https://user-images.githubusercontent.com/1216518/42421985-11293818-82a4-11e8-822c-db9f5c28fe26.png)
![image](https://user-images.githubusercontent.com/1216518/42421992-183766f2-82a4-11e8-8e6e-c292f72f66cd.png)

#### Expected behavior
- No overplotting, no unexpectedly dense regions of markers.

**Comment by pkerpedjiev** [comment]
Option #1 is fixed in v1.1.2. Somewhere along the way this introduced an issue with the multivec track (https://github.com/hms-dbmi/higlass/issues/282).

Having a hard time reproducing option #2.

**Comment by alexander-veit** [comment]
This issue seems to be fixed (tested with v1.7.2).
